src/db/scopes/product.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function applyScopes(sequelize) {
+  const { Product, ProductCategory, Extra } = sequelize.models
+
+  Product.addScope("categories", {
+    include: [
+      { model: ProductCategory, as: "categories", through: { attributes: [] } },
+    ],
+  })
+
+  Product.addScope("extra", {
+    include: [{ model: Extra, as: "extra" }],
```

Comment:
Add `through: { attributes: [] }` here like with categories.

This prevents the join table from being included as a child of every extra object, so it looks cleaner.